dhall/dhall.cabal

```diff
@@ -247,25 +252,20 @@ Common common
         uri-encode                                 < 1.6 ,
         vector                      >= 0.11.0.0 && < 0.13
 
+    GHC-Options: -Wall -Wcompat -Wincomplete-uni-patterns
+
     if flag(with-http)
       CPP-Options:
         -DWITH_HTTP
       if flag(use-http-client-tls)
         CPP-Options:
           -DUSE_HTTP_CLIENT_TLS
+      if flag(network-tests)
+        CPP-Options:
+          -DNETWORK_TESTS
 
-    GHC-Options: -Wall -Wcompat -Wincomplete-uni-patterns
-
-Library
-    Import: common
-    Hs-Source-Dirs: src
-    if impl(ghcjs)
-      Hs-Source-Dirs: ghcjs-src
-      Build-Depends:
-        ghcjs-base ,
-        ghcjs-xhr
-    else
-      Hs-Source-Dirs: ghc-src
+Common lib-depends
```

Comment:
What's the reason for splitting out `lib-depends` into a separate stanza from `common`?  The reason I ask is that it seems like the two are always used together anywa

Reply:
I think this was an artifact from when I was trying to sort out cabal-doctest.

Let me see if I can weed that out.